apollo-router-core/src/layers/headers.rs

```diff
@@ -224,6 +228,28 @@ struct PropagateService<S> {
     default_value: Option<HeaderValue>,
 }
 
+lazy_static! {
+    // Headers from https://datatracker.ietf.org/doc/html/rfc2616#section-13.5.1
+    // These are not propagated by default using a regex match as they will not make sense for the
+    // second hop.
+    // In addition because our requests are not regular proxy requests content-type, content-length
+    // and host are also in the exclude list.
+    static ref RESERVED_HEADERS: Vec<HeaderName> = [
```

Comment:
that could be an array instead of a vec initialized by lazy_static, no?

Reply:
+ 1